docs/.vitepress/Tabs.vue

```diff
@@ -1,44 +1,41 @@
-<template>
-  <div class="tab">
-    <header class="tab-header">
-      <ul class="tab-heads">
-        <li
-          class="tab-head"
-          v-for="tab in tabs"
-          :key="tab"
-          :class="{
-            'tab-head--active': state.activeTab === tab
-          }"
-          @click="switchTab(tab);"
-        >
-          <slot :name="`tab-head-${tab}`"></slot>
-        </li>
-      </ul>
-    </header>
-    <main class="tab-body">
-      <div class="tab-panel"> 
-      <slot :name="`tab-panel-${state.activeTab}`"></slot></div>
-    </main>
-    
-  </div>
-</template>
-
 <script setup>
-  import { reactive} from 'vue';
-
-  const props = defineProps({
-    initialTab: String,
-    tabs: Array
-  })
-  const state = reactive({activeTab: props.initialTab});
-  function switchTab(tab) {
-    state.activeTab = tab;
-  };
+import { ref } from 'vue';
 
+const props = defineProps({
+  initialTab: String,
+  tabs: Array
+});
+const activeTab = ref(props.initialTab);
+const switchTab = (tab) => {
+  activeTab.value = tab;
+};
 </script>
 
-<style scoped> 
+<template>
+<div class="tab">
```

Comment:
Not indentation inside <template> for <div>?

Reply:
There should be in the template yes... I fix. 👍